pyccel/ast/core.py

```diff
@@ -5155,7 +5155,7 @@ def bodies(self):
         return b
 
 
-class IfTernaryOperator(Basic, PyccelAstNode):
+class IfTernaryOperator(Basic, PyccelOperator):
```

Comment:
Could you move this operator to `ast.operators` please

Reply:
i almost want to suggest that.